### PR TITLE
#7 Create Reset Password Page

### PR DESCRIPTION
**Pull request by Sillor**
### **What's done**:
- Created the "Reset Password" page
- Made it responsive
- Added a few active effects on the button and link
- Created a header

It still requires success and error messages, but we will add them when the backend is implemented so that they will be displayed after fetching. Also will have to change the link reference based on the Sign up page address.

### CODE REVIEW

File: src/layout/ResetPassword.css
Comment (lclemente225) [inline src/layout/ResetPassword.css:1]: Question: I noticed that the body is selected to have the background image. Will all pages be using that background image? If not, could it be placed in a class that could be inserted into the parent element for the components that need it?

Comment (Sillor) [reply src/layout/ResetPassword.css:1]: Moved background, fonts and sizes to App.css

File: src/components/Header.jsx
[the remark this answers is not in the document]
Comment (Sillor) [reply src/components/Header.jsx:1]: Changed my navbar to the new one

**Review by AlissonRoss — APPROVED** [review]
Looking good

**Review by wyzhaobvt — APPROVED** [review]
Good job!

**Pull request merged**